avocado/plugins/spawners/podman.py

```diff
@@ -422,12 +434,21 @@ async def update_requirement_cache(
             "commit", "-q", runtime_task.spawner_handle
         )
         image_id = stdout.decode().strip()
+        tag_prefix = self.config.get("spawner.podman.image_tag_prefix")
```

Comment:
What about moving the image creation into its own private method? IMO, it would increase the readability of code, because it seems strange to me that half of the code of `update_requirement_cache` method is about creating podman images. What do you think?